libs/pages/elewa/social-impact/src/lib/components/social-impact/social-impact.component.ts

```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'elewa-group-social-impact',
+  templateUrl: './social-impact.component.html',
+  styleUrls: ['./social-impact.component.scss'],
+})
+export class SocialImpactComponent {}
```

Comment:
SocialImpactPageComponent is the main page in pages/

This issue should be a component under different name i.e SocialImpactBusinessComponent